composer.json

```diff
@@ -11,49 +11,56 @@
   ],
   "autoload": {
     "psr-4": {
-      "Shopping\\ApiTKDeprecationBundle\\": ""
-    },
-    "exclude-from-classmap": [
-      "Tests/"
-    ]
+      "Shopping\\ApiTKDeprecationBundle\\": "src/"
+    }
+  },
+  "autoload-dev": {
+    "psr-4": {
+      "Shopping\\ApiTKDeprecationBundle\\Tests\\": "tests/"
+    }
   },
   "repositories": [
     {
       "type": "vcs",
-      "url": "https://github.com/byWulf/apitk-common-bundle.git"
+      "url": "https://github.com/alexdo/apitk-common-bundle.git"
     },
     {
       "type": "vcs",
-      "url": "https://github.com/byWulf/apitk-header-bundle.git"
+      "url": "https://github.com/alexdo/apitk-header-bundle.git"
```

Comment:
Both VCS-URLs need to be changed back as soon as their respective PRs are tagged and released


- [ ] https://github.com/byWulf/apitk-header-bundle/pull/9
- [ ] https://github.com/byWulf/apitk-common-bundle/pull/14